### ISSUE TITLE
[ALM Accelerator - Feature]: Overwrite customizations shouldn't be default true

### ISSUE THREAD

**Issue by SabrinaDiBartolomeo** [microsoft/coe-starter-kit#4975]
### Describe the issue

The default in the ALM Accelerator Pipelines is to overwrite customizations. This should be default false because "Maintain customizations" is the recommended option.  By manual import this option is not available anymore.

The default for the Overwrite flag is specified here in the pipeline templates:
https://github.com/microsoft/coe-alm-accelerator-templates/blob/0947facac42e4cf0f4f3b046adea5234d9832d3f/Pipelines/Templates/deploy-Solution.yml#L16

 

### Expected Behavior

 The default in the ALM Accelerator Pipelines should be not to overwrite customizations

### What component are you experiencing the issue with?

ALM Accelerator Pipelines

### What solution version are you using?

latest

AB#1047

**Comment by devkeydet** [comment]
"recommended option" is debatable.  Otherwise, it wouldn't be an option.  We'll discuss as a team on how to give users more flexibility in choosing what option to use based on their preference.  Many people want to overwrite customizations in a CI/CD scenario to have somewhat of an auto healing mechanism.  Meaning that if someone manually edits things in a target environment when they are not supposed to, and wreaks havoc in the environment, the team building the app can rely on the fact that source control is the source of truth and the next deployment automation will "heal" the human intervention that caused issues.

**Comment by devkeydet** [comment]
Changed title to reflect this is not a bug but an enhancement request.  Also relabeled.

**Comment by SabrinaDiBartolomeo** [comment]
I agree with you. It depends. It would be nice to have the possibility to set this Option from UI or with pipeline variable. Thank you.

**Comment by AmarSaiRam** [comment]
According to https://learn.microsoft.com/en-us/power-platform/alm/pipelines, Power Platform Pipelines are now the recommended approach for Application Lifecycle Management (ALM). The ALM Accelerator is no longer actively maintained or supported. Therefore, this issue is being closed accordingly.
